### PR TITLE
Version 2.0.0-dev.3.10

### PR DESCRIPTION
**Pull request by lexaknyazev**
### New Features

* Added support for `KHR_materials_anisotropy` extension.
  * Added `KHR_MATERIALS_ANISOTROPY_ANISOTROPY_TEXTURE_TEXCOORD` validation warning.

* Added support for `KHR_materials_dispersion` extension.
  * Added `KHR_MATERIALS_DISPERSION_NO_VOLUME` validation warning.

* Added new tangent space validations: `MESH_PRIMITIVE_NO_TANGENT_SPACE` validation error and `MESH_PRIMITIVE_GENERATED_TANGENT_SPACE` validation warning.

* Added new `KHR_MATERIALS_CLEARCOAT_CLEARCOAT_NORMAL_TEXTURE_TEXCOORD` validation warning.

* Added partial support for the `KHR_animation_pointer` extension.
  * Added new `KHR_ANIMATION_POINTER_ANIMATION_CHANNEL_TARGET_NODE` and `KHR_ANIMATION_POINTER_ANIMATION_CHANNEL_TARGET_PATH` validation errors.
  * Added new `INCOMPLETE_EXTENSION_SUPPORT` validation info.

* Web version now supports external resources in directories.

### Changes

* Removed `UNRESERVED_EXTENSION_PREFIX` validation warning.

* Removed `KHR_MATERIALS_IRIDESCENCE_THICKNESS_RANGE_INVALID` validation error following the spec update.

### Bugfixes

* Fixed incorrect errors when an indexed attribute, e.g., `COLOR_0`, is used in morph targets (#178, #213).

* Fixed incorrect errors in the Web version when an asset contains non-relative URIs (#229).

* Fixed crash when skin weights data is missing (#196).

### Integration updates

* The `validateBytes` JS API function now supports a `format` validation option to bypass format detection and force GLB or glTF JSON explicitly.

* Added an option to ignore all validation issues except those passed explicitly; supported in both JS API and YAML config file.

### CODE REVIEW

**Review by emackey — APPROVED** [review]
I tested the NPM build of this with gltf-vscode, and everything seems good.

Mostly for my own reference: I had some odd issue possibly caused by leftover stuff in my dart cache. @lexaknyazev fixed it with the following suggestion:

1. Remove the `.dart_tool` directory and the `pubspec.lock` file.
2. Call `dart pub cache clean --force`
3. Call `dart pub get`
4. Try building again

I'm not sure what happened there but the tool is working well now.  Looks good!

**Comment by lexaknyazev** [comment]
@emackey Updated the README with these steps and other useful information.

File: README.md [hunk 1]
```diff
@@ -82,48 +83,86 @@ Shell return code will be non-zero if at least one error was found.
...
 To generate [ISSUES.md](ISSUES.md), follow these steps after installation:
+
 1. Run `dart run grinder issues`.
 2. `ISSUES.md` file will be written to the repo root.
+
+## Contributing
+
+Before making a pull request do the following steps:
+
+1. Run `dart format .` (note the dot) to format all source files.
```
Comment (emackey) [inline README.md:165]: We lost a parameter here, `dart format --fix .`  Was that intentional?

**Review by emackey — APPROVED** [review]
Looks good.  One question below.

File: README.md (hunk 1 above)
Comment (lexaknyazev) [reply README.md:165]: Yes, that option should not be used in regular workflow.

**Comment by emackey** [comment]
Published to NPM: https://www.npmjs.com/package/gltf-validator

/cc @donmccurdy @bghgary @elalish

**Comment by lexaknyazev** [comment]
The [live drag-n-drop tool](https://github.khronos.org/glTF-Validator) has been updated.

**Comment by donmccurdy** [comment]
Thanks! Updates for glTF Transform:

- https://github.com/donmccurdy/glTF-Transform/pull/1547

**Pull request merged**